docs/file_explorer.md

```diff
@@ -1,68 +1,108 @@
 File Explorer
 =============
+The File Explorer is for managing Workspaces and files.  There is also a context menu that pops up when you right click on a file or folder.
 
 To get to the File Explorer module - click the File Explorer icon.
 
-![](images/a-file-explorer1.png)
+![](images/a-file-explorer1a.png)
 
-The File Explorer is for managing Workspaces and files.  There is also a context menu that pops up when you right click on a file or folder.
+The green checkmark at the top of the page means that this plugin is in the Remix codebase.  When you click on the caret it will show more info about the plugin including a link to this documentation.
+
+![](images/a-fe-top-caret.png)
 
 File Storage
 ------------
 By default, Remix IDE stores files in **IndexedDB**.  
 
-Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
+Coding in Remix IDE Online is different from writing in a Google doc. 
+- A Google doc saves your work to your account on Google’s servers.
+- Remix has no user accounts.  By default, files are ONLY saved locally in the browser’s storage. So tread carefully, browser storage is not permanent!
+
+See below for using other storage locations - notably **Remixd** to connect Remix to a specific folder on your hard drive.
 
 **Important Note:** Clearing the browser storage & IndexedDB will **permanently delete** all the files stored there. 
 
-## File Storage outside of the browser and Version Control
+### File Storage on your hard drive
+If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix IDE](https://github.com/ethereum/remix-desktop/releases/)**. 
+- Remixd is a script that you run on your computer that shares a specified folder on your computer with Remix IDE.
+- Remix Desktop is a version of Remix IDE in an Electron app. When using Remix Desktop, you cannot use a browser wallet like MetaMask, because Remix Desktop does not run in a browser.  You'd need to connect using a service like Infura.
+
+### Connecting Remix to remote Git repos
+**If you are not using Remixd, it is highly recommended that you save to a remote repo.** 
+(Browsers do crash causing localstorage & indexedDB to be corrupted)
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
+Remix IDE can be connected to remote Git repos both on services like GitHub or repos in IPFS.  
+Most of the Git operations are done in the **DGIT** pluin. (DGIT stands for Decentralized GIT).
```

Comment:
I think filip refrain to use the word `Decentralized GIT`

Reply:
We have had users ask what DGIT means, so its worth it to leave it.